src/AutoRest.CSharp/Common/Generation/Writers/FormattableStringHelpers.cs

```diff
@@ -90,6 +90,18 @@ public static FormattableString GetConversionFormattable(this Parameter paramete
                 return $"{typeof(RequestContent)}.{nameof(RequestContent.Create)}({parameter.Name})";
             }
 
+            if (parameter.Type.IsEnum && toType.EqualsIgnoreNullable(typeof(RequestContent)) && parameter.Type.Implementation is EnumType enumType)
```

Comment:
The implementation here has the same issue as I mentioned [here](https://github.com/Azure/autorest.csharp/pull/3289#discussion_r1172095705)